packages/vue/src/components/atoms/SfImage/SfImage.stories.js

```diff
@@ -50,14 +63,88 @@ storiesOf("Atoms|Image", module)
     },
     template: `<SfImage
         :src="src"
+        :srcset="srcset"
         :alt="alt"
         :width="width"
         :height="height"
         :lazy="lazy"
-        :picture-breakpoint="pictureBreakpoint" 
         :rootMargin="rootMargin"
         :threshold="threshold"/>`,
   }))
+  .add("With srcset as string", () => ({
```

Comment:
I found 2 errors in console showing regarding this story that suggest some srcset issue: 
`Failed parsing 'srcset' attribute value since it has an unknown descriptor.`
`vue.esm.js:6796 Dropped srcset candidate "[object"`

Reply:
oh, right, thanks :)